.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+    - "3.6"
```

Comment:
I think we can trust that the Python devs won't break backward compatibility between minor updates, so I'm not sure if testing on Python 3.6 is needed.